golos.social/golos.social.cpp

```diff
@@ -113,5 +116,44 @@ void social::unblock(account_name blocker, account_name blocking) {
     });
 }
 
+void social::setreput(account_name voter, account_name author, base_t subtract, base_t add) {
```

Comment:
Using a parameter `subtract` looks like a security bug. It is necessary to transmit only the value by which the reputation should be changed.
I would suggest to call this method `changereput`. It will be closer in meaning.

Reply:
Yes. Reputation should reflect the first reaction of the user.